src/Features/Core/Portable/Completion/CompletionOptionsProvider.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System.Collections.Generic;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Wow. I thought @jaredpar's formatter thing checked for this. Have we stopped running that? (cc: @tannergooding)